code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -22,11 +22,11 @@ makeMd = vcat . punctuate secSep . filtEmp
 introInfo :: String -> [String] -> Maybe String -> Doc
 introInfo name auths descr = introSec (text name) (listToDoc auths) (length auths) (maybePurpDoc descr)
 
--- | Instruction section, contains 3 paragraphs, Running, Building and Config Files.
+-- | Instruction section, contains 4 paragraphs, Running, Building, Input-Output and Config Files.
 -- The Config file section is only displayed if there are configuration files.
-instDoc :: [String] -> Doc
-instDoc cfp = regularSec (text "Making Examples") 
-    (runInstDoc <> doubleSep <> makeInstDoc) <> configSec cfp 
+instDoc :: [String] -> String -> Maybe String ->  Maybe String -> Doc
+instDoc cfp name inf outf = regularSec (text "Making Examples") 
+    (runInstDoc <> doubleSep <> makeInstDoc) <> inOutFile name inf outf <> configSec cfp
```

Comment:
If `inf == Nothing`,`runInstDoc` still includes the instructions `make run RUNARGS=input.txt`, even when there is no `input.txt`; this should be updated.

Reply:
Currently, we have no alternative for this, should this simply be not generated in cases of input being empty for now?

Reply:
Yes, or you can come up with an alternative message to be displayed

Reply:
Personally, I think that this particular PR maybe should not have the `Maybe` for input or output, and simply focus on refactoring where the information comes from. A subsequent PR can implement a proper design for different architectures that may make some input/output optional.